File: app/src/main/java/info/dafle/testeandroid/service/ApiService.java
package info.dafle.testeandroid.service;

import info.dafle.testeandroid.model.ContatoResponse;
import info.dafle.testeandroid.model.Fund;
import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {

    @GET("/cells.json")
    Call<ContatoResponse> getCells();

    @GET("/fund.json")
    Call<Fund> getFund();
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/system/retrofit/ApiService.kt
package andcordeiro.com.br.testeandroid.system.retrofit

import andcordeiro.com.br.testeandroid.entities.ResultContact
import andcordeiro.com.br.testeandroid.entities.ResultScreen
import retrofit2.Call
import retrofit2.http.GET

interface ApiService {

    @GET("cells.json")
    fun getCells(): Call<ResultContact>

    @GET("fund.json")
    fun getScreen(): Call<ResultScreen>
}